src/redux/categories/categories.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <h2>This page is under construction</h2>;
+
+export default Categories;
```

Comment:
- [ ] This is a Categories component and so it is redundant since you already defined the main component in `src/pages/categories.js`. At this point in the project, this file should be empty and will only be used for redux-related actions. Kindly delete these lines.

Reply:
Done! Thanks!